### PR TITLE
Updated the Chapter and Venue forms to be consistent with other forms.

### LINKED ISSUE #800: Chapter and Venue forms need updating, including validation and placeholders
They currently look like this:
image

while the Event form is more polished:
image

The look and feel should be consistent, with the Event form being the template for the other two.

### PR DESCRIPTION
**Pull request by Ravichandra-C**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

Closes #800 

1. Updated Chapter and Venue forms to be consistent with other forms.
2. Added Place holders to the forms.
3. Removed the details fields from Chapter form, Chapter Schema , Input types and resolvers

### CODE REVIEW

**Comment by Ravichandra-C** [comment]
@ojeytonwilliams @allella , Could you please take a look at the placeholders used in the forms. I tried to use freecodecamp as an example chapter and wrote place holders depending on that in the forms. Is that ok?

File: client/src/modules/dashboard/Chapters/components/ChapterForm.tsx [hunk 1]
```diff
@@ -16,20 +17,65 @@ interface ChapterFormProps {
...
+    label: 'Description',
+    placeholder:
+      'freeCodeCamp is a non-profit organization that helps people learn to code for free',
+    required: true,
+    type: 'textarea',
+  },
+  {
+    key: 'city',
+    label: 'City',
+    placeholder: 'San Fransisco',
```
Comment (ojeytonwilliams) [inline client/src/modules/dashboard/Chapters/components/ChapterForm.tsx:47]:
```suggestion
    placeholder: 'San Francisco',
```

File: client/src/modules/dashboard/Chapters/components/ChapterForm.tsx [hunk 2]
```diff
@@ -16,20 +17,65 @@ interface ChapterFormProps {
...
+    label: 'Chapter name',
+    placeholder: 'freeCodeCamp',
+    required: true,
+    type: 'text',
+  },
+  {
+    key: 'description',
+    label: 'Description',
+    placeholder:
+      'freeCodeCamp is a non-profit organization that helps people learn to code for free',
```
Comment (ojeytonwilliams) [inline client/src/modules/dashboard/Chapters/components/ChapterForm.tsx:40]:
```suggestion
      'freeCodeCamp is a nonprofit organization that helps people learn to code for free',
```
Consistency with 'category' below.

File: client/src/modules/dashboard/Chapters/components/ChapterForm.tsx [hunk 3]
```diff
@@ -16,20 +17,65 @@ interface ChapterFormProps {
...
+    key: 'category',
+    label: 'Category',
+    placeholder: 'Education and nonprofit work',
+    required: true,
+    type: 'text',
+  },
+  {
+    key: 'imageUrl',
+    label: 'Image Url',
+    placeholder: 'www.freecodecamp.com',
```
Comment (ojeytonwilliams) [inline client/src/modules/dashboard/Chapters/components/ChapterForm.tsx:75]:
```suggestion
    placeholder: 'www.freecodecamp.org',
```
Nitpick, .com just redirects to .org

File: client/src/modules/dashboard/Venues/components/VenueForm.tsx [hunk 4]
```diff
@@ -13,20 +13,79 @@ interface VenueFormProps {
...
+    key: 'street_address',
+    label: 'Street address',
+    placeholder: 'Street address',
+    isRequired: true,
+    type: 'text',
+  },
+  {
+    key: 'city',
+    label: 'City',
+    placeholder: 'San fransisco',
```
Comment (ojeytonwilliams) [inline client/src/modules/dashboard/Venues/components/VenueForm.tsx:44]:
```suggestion
    placeholder: 'San Francisco',
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
@Ravichandra-C I think those are good placeholders, I just found a few typos/inconsistencies.

The look and feel is way better, though.

**Comment by ojeytonwilliams** [comment]
@Ravichandra-C sorry about the failing CI, I pushed changes before merging the updates to graphql-codegen :man_facepalming:  They should resolve themselves next commit, but if they don't I'll fix it.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Pull request merged**